### ISSUE TITLE
Menu contextuel Commands ( Apps in Menu )

### ISSUE THREAD

**Issue by proceau** [discord-php/DiscordPHP#691]
Extend interactive/command class to Apps Menucontext

https://discord.com/developers/docs/interactions/application-commands#application-command-object

it's https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-types

Type USER & MESSAGE

And a exemple in sample :)

**Comment by SQKo** [comment]
Context menu application command is already available on v7.0.0-RC2 or `dev-master` branch, you can register your command with type [User](https://github.com/discord-php/DiscordPHP/blob/ee749fcd62c45aa7d8e0b00f30ad78ae5b7baeff/src/Discord/Parts/Interactions/Command/Command.php#L44) or type [Message](https://github.com/discord-php/DiscordPHP/blob/ee749fcd62c45aa7d8e0b00f30ad78ae5b7baeff/src/Discord/Parts/Interactions/Command/Command.php#L47)
then use [`listenCommand`](https://github.com/discord-php/DiscordPHP/blob/ee749fcd62c45aa7d8e0b00f30ad78ae5b7baeff/src/Discord/Discord.php#L1536) using the name of the command.

Since it's still in RC stage, documentation is being prepared.

Example, register guild message context menu command 'Bookmark':
```php
$commandbuilder = CommandBuilder::new()
->setName('Bookmark')
->setType(Command::MESSAGE);

$command = $guild->commands->create($commandbuilder->toArray());

$guild->commands->save($command);
```

To listen the command interactions:
```php
$discord->listenCommand('Bookmark', function (Interaction $interaction) use ($discord) {
    $interaction->respondWithMessage(MessageBuilder::new()->setContent('Message bookmarked!');
});
```
You can customize your interaction code as you wish, browse the [Interaction documentation](https://discord-php.github.io/DiscordPHP/#interactions) and [Message builder here](https://discord-php.github.io/DiscordPHP/#message-builder)

**Comment by proceau** [comment]
Perfect SQKo i'm complet the code.
But i'm : "message": "Missing Access",     "code": 50001
but m'y bot it's "administator" on discord, what this "access" ?
I'm read API but it's not clear
